packages/api/src/index.ts

```diff
@@ -44,3 +44,6 @@ export const getSchema = async (options: Options) => {
 
   return directives.reduce((s, d) => d.transformer(s), schema)
 }
+
```

Comment:
It seems that you're exporting only the QueryProduct root and resolver. Shouldn't you be exporting ALL possible queries/types/resolvers that we have?

Reply:
That's the idea. I've created a task in our backlog so that we can export all Root Objects in the starter and leverage the use of TS. Also, I think we should re-export this types from the core so that we don't need @faststore/api in the starter project.